tests/unittests/parsing/PreprocessorTests.cpp

```diff
@@ -1326,6 +1326,30 @@ TEST_CASE("unconnected_drive directive") {
     CHECK(!diagnostics.empty());
 }
 
+TokenKind lexCellDefine(std::string_view text) {
+    diagnostics.clear();
+
+    Preprocessor preprocessor(getSourceManager(), alloc, diagnostics);
+    preprocessor.pushSource(text);
+
+    Token token = preprocessor.next();
+    REQUIRE(token);
+    return preprocessor.getCellDefine();
+}
+
+TEST_CASE("celldefine directive") {
+    CHECK(lexCellDefine("`celldefine") == true);
+    CHECK(lexCellDefine("`celldefine\n`endcelldefine") == false);
+    CHECK(lexCellDefine("`endcelldefine") == false);
+    CHECK_DIAGNOSTICS_EMPTY; // TODO: is this right? lexCellDefine() does diagnostics.clear()
```

Comment:
Yeah, you'd need this after each lexCellDefine call.

Reply:
There are many other calls to `lex*` like this elsewhere in the file with the same issue; should they be fixed too?

Reply:
Hmm yeah, looks like the unconnected drive and timescale tests have the same problem.

Reply:
Fixed those.